app/javascript/src/components/Dashboard/Notes/DropMenu.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { MenuVertical } from "neetoicons";
+import { Dropdown } from "neetoui";
+
+const DropMenu = ({ onClick }) => {
+  const { Menu, MenuItem } = Dropdown;
+
+  return (
+    <Dropdown customTarget={<MenuVertical size={17} />}>
+      <Menu>
+        <MenuItem.Button onClick={onClick}>Edit</MenuItem.Button>
+        <MenuItem.Button>Delete</MenuItem.Button>
```

Comment:
Go through the Slack thread where I have put up review comments.

Reply:
Ok, I will change the `onClick` to `handleDropdown` and add `style` as `danger`.